yapapi/runner/events.py

```diff
@@ -0,0 +1,117 @@
+"""Code for representing and logging events in Golem computation."""
```

Comment:
okay, nitpicking but saying "code" here is kinda superfluous as everything here is code ;) ... maybe it should say something like "mechanism"? ...

Reply:
Changed this to `"""Representing and logging events in Golem computation."""` :)